sat/cli/bootsys/cron.py

```diff
@@ -57,6 +58,8 @@ def modify_cron_job(ssh_client, hostname, job_file_path, enabled=False):
             or disable the cron job, or if the cron job should be enabled but the
             given file path does not exist
     """
+    # Set the policy to automatically add unknown hosts to known_hosts
+    ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
```

Comment:
If we are already setting the missing host key policy in the `get_ssh_client` function, then we should not need to do it again here. This function is called by `modify_ensure_ceph_mounts_cron_job`, which is called by `do_ceph_unmounts`, which is called by `do_mgmt_shutdown_power`. That's where the `SSHClient` instance is created by a call to `get_ssh_client`, and that function already calls `set_missing_host_key_policy` on the `ssh_client`.

Reply:
Thanks @haasken-hpe. addressed